pinaxcon/proposals/models.py

```diff
@@ -54,3 +54,53 @@ class MiniconfProposal(ProposalBase):
 
     class Meta:
         verbose_name = "miniconf proposal"
+
+class SysAdminProposal(Proposal):
+
+    TYPE_SHORT_PRESENTATION = 1
+    TYPE_LIGHTNING_TALK = 2
+    
+    TALK_FORMATS = [
+        (TYPE_SHORT_PRESENTATION, "Short Presentation (20 min)"),
+        (TYPE_LIGHTNING_TALK, "Lightning Talk (5 min)"),
+    ]
+    
+    talk_format = models.IntegerField(choices=TALK_FORMATS)
+    
+    class Meta:
+        verbose_name = "System Administration Miniconf Proposal"
+
+class WriteTheDocsProposal(Proposal):
+
+    TYPE_LONG_PRESENTATION = 1 
+    TYPE_SHORT_PRESENTATION = 2
+    
+    TALK_FORMATS = [
+        (TYPE_LONG_PRESENTATION, "Long Presentation (40 min)"),  
+        (TYPE_SHORT_PRESENTATION, "Short Presentation (20 min)"),
+    ]
+    
+    talk_format = models.IntegerField(choices=TALK_FORMATS)
+    
+    class Meta:
+        verbose_name = "WriteThe Docs Miniconf Proposal"
+
+class OpenRadioProposal(ProposalBase):
```

Comment:
The proposal kinds make a slug and the slugs are matched to forms in settings.py and the forms are linked to a proposal model.

I was going to just use the base model, but this way in the /admin each miniconf gets it's own table so it's easy to find.